### DISCUSSION TITLE
Could not find frealign statistics file in maps folder

### DISCUSSION THREAD

**Discussion by ariel234156** [nextpyp/.github#118] [Q&A] [answered]
Hi,
When I run a particle refinement process, I get this warning:
"Could not find frealign statistics file in maps folder."
After this warning, the refinement continued regularly and finished without other errors.
I want to ask why this warning happened and if there is an influence on the final result?
thank you.

**Comment by abartesaghi** [comment]
The frealign statistics file is used to do frequency weighting during refinement according to the ssnr table from the previous iteration. If the file is missing, no weighting will be applied. So, yes, this will influence the result. What iteration number does this happen for? Does it happen during 3d classification?

**Reply by ariel234156** [reply]
I got this error in each iteration of particle refinement. Can this happen because I skip the ssnr table and check the "skip refinement" option when I run this step?

**Reply by abartesaghi** [reply, marked as answer]
We recently discovered a problem with how nextpyp handles the name of the frealign statistics file which likely caused this warning. When the option `Use SSNR table` is selected, the statistics file was correctly being copied to the local scratch folder but with the wrong name, which caused the ssnr table not to be used during refinement. That being said, frequency weighting during constrained refinement has relatively low influence in the final result, at least in our experience. We will fix in the next version, thanks for reporting!
